ui/src/components/NewWorkflow/Add.tsx

```diff
@@ -54,7 +42,13 @@ const useStyles = makeStyles({
   },
 })
 
-const types = ['single', 'serial', 'parallel', 'suspend', 'custom']
+export enum RenderableTemplateType {
```

Comment:
You could see that I introduce `HTTP` as `RenderableTemplateType`. In the definition from the backend, the type of workflow's `TemplateType` only includes `chaos`, `serial`, `parallel`, `suspend`, and `task`. `HTTP` is also a kind of `Task`. I think it's better to distinguish them. 

I think we should move all these related models to `ui/src/api`, because there are same definition in backend, and use some code generation tools for syncing them.